File: BookingApp.API/Models/Review.cs
using System;

namespace BookingApp.API.Models
{
    public class Review
    {
        public int Id { get; set; }
        public string Url { get; set; }
        public string Description { get; set; }
        public DateTime DateAdded { get; set; }
        public bool IsApproved { get; set; }
        public string PublicId { get; set; }
        public int Rating { get; set; }
        public User User { get; set; }
        public int UserId { get; set; }

        // From here //
        public Booking Booking { get; set; }
        public int BookingId { get; set; }
        // To here //
    }
}

File: BookingApp-SPA/src/app/_models/review.ts
import { FileItem } from 'ng2-file-upload';
import { Booking } from './booking';
import { User } from './user';

export interface Review {
    id?: number;
    url?: string;
    description: string;
    dateAdded?: Date;
    isApproved?: boolean;
    publicId?: string;
    rating: number;
    user?: User;
    userId?: number;
}
